packages/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 3.2.0
+Bug fixes:
+    * #717 - Old version of connectivity_plus package
```

Comment:
This is not the latest version of the file, flutter is already at version 3.1.2. I think you need to do a rebase (not just a branch merge) to fix these inconsistencies. And could you please take a look at the new changelog format.